### ISSUE TITLE
Log command fails in Etcd Operator scenario

### ISSUE THREAD

**Issue by mamurak** [openshift-labs/learn-katacoda#1278]
**In which scenario did this bug occur?**

Etcd Operator 

**At what step of the scenario did this happen?**

Step 3: Creating the Etcd Operator Deployment

**Describe the bug**

After creating the Etcd operator deployment, the user is instructed to follow the logs of the created pod:
```
export ETCD_OPERATOR_POD=$(oc get pods -l name=etcd-operator -o jsonpath='{.items[0].metadata.name}')
oc logs $ETCD_OPERATOR_POD -f
```
However, the command fails due to missing credentials.

**Logs or error messages**

`error: You must be logged in to the server (the server has asked for the client to provide credentials ( pods/log etcd-operator-84cf65cb74-mv2lz))`

**Comment by madorn** [comment]
Please see https://github.com/openshift-labs/learn-katacoda/issues/1276

**Comment by madorn** [comment]
Fixed in #1289

**Comment by jdob** [comment]
Closing per Matt's comment.
